### ISSUE TITLE
Deploy to Heroku as a Django project

### ISSUE THREAD

**Issue by athityakumar** [prachi1210/starbot#24]
If this project is deployed to Heroku as a Django project - it'd help even more GitHubbers as they wouldn't have to manually clone the whole repository, and run the scripts.

**Comment by jdleo** [comment]
to me this doesn't really seem to help "githubbers" but more of the noobs/script kiddies who are googling to find this stuff

**Comment by athityakumar** [comment]
@jdleo : Um, irrespective of how much ever talented of a programmer you might be, you'll always want to take the shortest solution to a problem. So, this web interface would be much preferred by GitHubbers who want quick stars, rather than cloning and running locally.

And you really have to find out if those who want quick solution are "kiddies" / "noobs" or just every single homo sapien. IMHO, EVERYONE will prefer the quicker solution.

**Comment by jdleo** [comment]
@athityakumar I know bro, I get what you're saying :smiley: . The fact that you even know what django/heroku are proves that you are not one of the people I'm referring to. I definitely wouldn't complain about a web version for it, just pointed out that this would enable the "noobs".

**Comment by athityakumar** [comment]
> to me this doesn't really seem to help "githubbers"

GitHubbers aren't just a collection of pro-programmers, GitHub includes noobs too. Basically, every pro started as a noob. So, it would easier for the noob self to just use the Django webapp, and for the pro self to contribute to this project.

Peace! :smile:

**Comment by athityakumar** [comment]
@prachi1210 @jdleo : Hmm, so - do you guys think that this issue should be implemented or should this issue be closed?

**Comment by prachi1210** [comment]
@athityakumar 
I feel there's no harm in going ahead with it. So if you want to work on it, feel free to.

**Comment by sahildua2305** [comment]
I agree with @jdleo totally. I strongly think that putting this project on Heroku will make it easily accessible to wrong-doers. As a GitHub project it's still restricted (to some extent).

**Comment by athityakumar** [comment]
@sahildua2305 : I don't really think I understand the term "wrong-doers". Isn't this project supposed to help get stars on repositories? So, why not make a Heroku App out of it, and make it easier for everyone? I hope this isn't about the "ethical" / "moral" value of allowing everyone to use this project with just one click (Heroku Django App).
